kicad_pcb.py

```diff
@@ -8,7 +8,7 @@
 A usage demonstration is available in `test.py`
 '''
 
-from .sexp_parser import *
+from sexp_parser.sexp_parser import *
```

Comment:
I should have made my comments more contextual. What I meant is to change this statement, not the one in `test.py`. Because `test.py` is meant to be run directly as a stand alone script, using absolute import there is actually fine. However, `kicad_pcb.py` here is meant to be used by other modules. With your current changes here, I can no longer import `kicad_parser` module (see [here](https://github.com/realthunder/fcad_pcb/blob/d8e0f8da6ea5f9718bf60ae8a7dfe8a6e395661b/kicad.py#L17)), without having error like below,

```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/home/thunder/.FreeCAD/Macro/fcad_pcb/kicad.py", line 17, in <module>
    from .kicad_parser import KicadPCB,SexpList,SexpParser,parseSexp
  File "/home/thunder/.FreeCAD/Macro/fcad_pcb/kicad_parser/__init__.py", line 1, in <module>
    from .kicad_pcb import *
  File "/home/thunder/.FreeCAD/Macro/fcad_pcb/kicad_parser/kicad_pcb.py", line 15, in <module>
    from sexp_parser.sexp_parser import *
ModuleNotFoundError: No module named 'sexp_parser'
```

Reply:
I thought your comment was about `kicad_pcb.py`, but it did not work. So I put it in the test.py where it worked on a test right after. Let me try to fix this. Yesterday I did not have too much time.